src/Layouts/Partials/LayersPanel.vala

```diff
@@ -236,12 +197,11 @@ public class Akira.Layouts.Partials.LayersPanel : Gtk.ListBox {
     public void reload_zebra () {
         loop = 0;
 
-        @foreach (row => {
-            if (!(row is Akira.Layouts.Partials.Artboard)) {
-                zebra_layer ((Akira.Layouts.Partials.Layer) row);
-                return;
-            }
+        items_list.@foreach (row => {
+            zebra_layer ((Akira.Layouts.Partials.Layer) row);
+        });
 
+        artboards_list.@foreach (row => {
             zebra_artboard ((Akira.Layouts.Partials.Artboard) row);
         });
```

Comment:
Here it would be nice to only reload the zebra of the list that was changed, and handling separated zebras per artboards, but that's something we can deal in future PRs.

Reply:
Yes, I agree on updating only the artboard that has changed.

Btw, each artboard is treated as separated wrt zebra handling, so I don't get your question about treating them separately.

Reply:
Really?
The zebra coloring of my layers changes on all the artboards whenever I add a new layer.
Does it happen to you? As a general rule, the first layer inside an artboard should always have a white background.
I'll attach a screenshot.

Reply:
Ok I noticed the issue you reported and I have to admit that I've never been able to spot it, good catch!

I guess it's because your eye is not used as mine at staring at the Layers panels and simply checking layers get created and destroyed after user interaction, that's why is always good to have someone else to look at your code!

I think we could delay this to a future PR, but if you do not agree I could try to sort this out.

Reply:
Never mind, it has been easier than what I expected, I fixed the issue with 2 line (one is an explanatory comment :))